### DISCUSSION TITLE
How to kill (or close app) KeePassXC by scripting on Windows 10 (powershell/stop-Process or cmd/Taskkill.exe ?

### DISCUSSION THREAD

**Discussion by yoda65** [keepassxreboot/keepassxc#9950] [Q&A]
Hi
I need to kill the KeePassXC process by scripting on Windows 10 pro
With powershell (Stop-Process) or with the cmd "taskkill /PID"  command,
message "Access denied" is return (sometime it works but not very often and  I don't know in which case). I checked 
I tested several options (/force ...)

The manual process killing with windows Taskmanager works fine.

So what is the correct way to kill (or close) keepassxc by script?

Regards

**Comment by droidmonkey** [comment]
Access denied would indicate the account running the script is lower privilege than the account running KeePassXC.

**Reply by yoda65** [reply]
No.
I use keepassXC portable lauched manaully with a cmd script on my current user account (no admin privilege) . My "killing" scripts (powershell ou cmd) runs on the same current user account.
I do not understand what is happening. 

Regards
